test/index.js

```diff
@@ -27,11 +27,11 @@ test.before(async () => {
     path: pathToBinary
   });
 
-  await app.start();
+  return app.start();
```

Comment:
I don't see how using async/await here is over the top. It works pretty much the same internally (async functions return a promise), but is more readable and easier to work with. What if you suddenly want to do something after `app.start()`?